app/master/build_request_handler.py

```diff
@@ -148,7 +149,13 @@ def _compute_subjobs_for_build(self, build_id, job_config, project_type):
         :type project_type: project_type.project_type.ProjectType
         :rtype: list[Subjob]
         """
-        atoms_list = job_config.atomizer.atomize_in_project(project_type)
+        # Users can override the list of atoms to be run in this build. If the atoms_override
+        # was specified, we can skip the atomization step and use those overridden atoms instead.
+        if project_type.atoms_override is not None:
+            atoms_string_list = project_type.atoms_override
+            atoms_list = [Atom(atom_string_value) for atom_string_value in atoms_string_list]
```

Comment:
Ah, hmm -- yeah it might be cool if we did not require atoms to be output by the atomizer -- more flexibility, etc. I support leaving it how you currently have it and improving it later if it causes terribly confusing errors or behaviors.